tests/player.test.js

```diff
@@ -17,6 +17,17 @@ describe('Player:', function () {
         browser.getAttribute('#video-player', 'ended') !== 'true'
       ))
     })
+    browser.addCommand('waitUntilBuffered', () => {
```

Comment:
I am pretty sure this is the reason some player tests were failing inconsistently. If buffering had not started, and we tried to play the video in a test, the play state would seem to change but the video would not actually progress.